config/robots.rb

```diff
@@ -2024,4 +2024,109 @@
                    target_state: 'passed'
                  }
                })
+
+  # Robots for pWGS 384 pipeline
+
+  custom_robot('star-pwgs-384-post-shear-to-pwgs-384-post-shear-xp',
+               name: 'STAR pWGS-384 Post Shear => pWGS-384 Post Shear XP',
```

Comment:
I see this matches what it says on the Confluence page, but -
I think these names tend to be <robot> <input plate type> => <output plate type>
So I think it should be 'STAR LB Post Shear => pWGS-384 Post Shear XP' 

Make sense do you think?

Reply:
Fixed